docs/recipes/hyperlinks.md

```diff
@@ -0,0 +1,78 @@
+# Use hyperlinks directly in the terminal
+
+This recipe is an example on how to leverage hyperlinks directly in the terminal.
+Hyperlinks are generated by terminal applications by using the [OSC-8 escape sequences](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda).
+
```

Comment:
I think you should add another paragraph describing what the experience of having this setup is.
For example mentioning that when you (ctrl/super-)click on a folder path in ls output, the shell will jump to that directory. Etc..